magenta/interfaces/midi/README.md

```diff
@@ -147,61 +151,22 @@ using VPMK and FluidSynth, your command would look like this:
 $ bazel-bin/magenta/interfaces/midi/magenta_midi \
   --input_port="VMPK Output" \
   --output_port="FluidSynth virtual port" \
-  --bundle_file=/tmp/attention_rnn.mag
+  --bundle_file=/tmp/attention_rnn.mag \
+  --phase_bars=4
```

Comment:
I meant "phase" as in "call phase" vs "response phase". To remove the confusion, I've changed this to "phrase bars" and used the terms "call stage" and "response stage".